extensions.json

```
@@ -211,6 +211,11 @@
       "repository": "https://github.com/dsznajder/vscode-es7-javascript-react-snippets",
       "version": "2.8.0"
     },
+    {
+      "id": "dylan-foundry.vscode-dylan",
+      "repository": "https://github.com/dylan-lang/vscode-dylan",
+      "version": "0.2.1"
```

I think it's fine to publish this extension straight from the default branch without pinning a specific version.

```suggestion
      "repository": "https://github.com/dylan-lang/vscode-dylan"
```